bindings/wasm/bindings.js

```diff
@@ -95,4 +95,21 @@ Module.setup = function () {
     polygonsVec.delete();
     return result;
   };
+
+  function batchbool(name) {
+    return function(...args) {
+      if (args.length == 1)
+        args = args[0];
+      let v = new Module.Vector_manifold();
+      for (const m of args)
+        v.push_back(m);
+      const result = Module['_' + name + 'N'](v);
+      v.delete();
+    return result;
+    }
+  }
+
+  Module.union = batchbool('union');
+  Module.difference = batchbool('difference');
```

Comment:
How do you batch a difference operation? There's a negative and a positive object, so which is which with n?

Reply:
The same as openscad, the first one is positive and the remaining ones are negative.